xwiki-commons-tools/xwiki-commons-tool-xar/xwiki-commons-tool-xar-plugin/src/main/java/org/xwiki/tool/xar/XARMojo.java

```diff
@@ -215,6 +219,22 @@ private void performTransformations() throws Exception
                             }
                             break;
 
+                        case INSERT_ATTACHMENT_CONTENT:
+                            if(content != null) {
+                                node.selectSingleNode("filesize").setText(Long.toString(content.length(),10));
+                                node.selectSingleNode("content").setText( "\r\n" +
+                                    new String(
+                                        Base64.getMimeEncoder().encode(FileUtils.readFileToByteArray(content)),
```

Comment:
You might want to use `Base64.encodeBase64String()` instead.

Reply:
There's several of them. Only the one with getMimeEncoder() produces the line-broken version which is a key for somewhat readable commits. I've now removed the charset.